NEWS.md

```diff
@@ -0,0 +1,27 @@
+# data.validator 0.2.0
+
+-   `validate_cols()` and `validate_rows()` will use all columns in dataframe if no column is passed
+
+-   Added a defensive wrapper (`data.validator:::check_assertr_expression()`) to add evaluation error to regular validation errors
+
+-   New examples added:
+
+    -   Shiny App example demo example added
+    -   Usage examples added in README
+    -   Connect workflow examples added
+
+-   Removed deprecated `.data` calls in `dplyr::select()`
+
+-   `shiny.semantic::accordion()` bug fixes
+
+# data.validator 0.1.6
+
+-   Dependencies updated
```

Comment:
Please describe the purpose of this release, not a vague "dependencies update".

Reply:
Only package dependency change I was able to find in #25 was related to ROxygen version upgrade.  So I've added this change in NewsMD file.

Reply:
As I wrote in my previous comment: Version 0.1.6 was pushed to CRAN only to restore data.validator (after one of its dependencies was removed) and there were __no actual changes__ implemented. The changelog should reflect that - you need to write that there were __no changes__ and describe why (in a super condensed way). Simply read the comments in the PR you linked here to learn more (shiny.semantic was down and all packages that depended on it were also removed from CRAN).